src/persist-client/Cargo.toml

```diff
@@ -43,6 +43,7 @@ mz-ore = { path = "../ore", features = ["bytes_", "test", "tracing_"] }
 mz-persist = { path = "../persist" }
 mz-persist-types = { path = "../persist-types" }
 mz-proto = { path = "../proto" }
+mz-stash = { path = "../stash" }
```

Comment:
i would much prefer to not introduce this dependency! let's talk about how we might avoid it, please :)

i think the answer might be that ShardId already has `RustType` impl to serialize it to/from a String, which is the intended way to durably write down ShardIds. the`[u8; 16]` are an internal detail

Reply:
Totally makes sense! I updated the PR and removed the dependency from `persist-client`, using `String` as the protobuf type

Reply:
🙌 thanks!